Secrets/KeysExample.swift

```diff
@@ -1,6 +1,2 @@
-struct Keys {
-    struct Parse {
-        static let applicationId = ""
-        static let clientKey = ""
-    }
-}
+let x = 1
+let y = 1   
```

Comment:
Trailing Whitespace Violation (Medium Severity): Line #2 should have no trailing whitespace: current has 3 trailing whitespace characters